### PR TITLE
Feature/search results pages

### PR DESCRIPTION
**Pull request by MarcelUrbanczyk**
created switching pages with search results

### CODE REVIEW

File: src/features/SearchResults/SearchMovies/searchResultSaga.js
Comment (Soniq7) [inline src/features/SearchResults/SearchMovies/searchResultSaga.js:1]: I see searchResultSaga is deleted - was it not needed?

File: src/features/SearchResults/SearchPeople/searchResultSaga.js
Comment (Soniq7) [inline src/features/SearchResults/SearchPeople/searchResultSaga.js:1]: Same here - Saga for SearchPeople also not needed?

**Review by Soniq7 — APPROVED** [review]

File: src/features/SearchResults/SearchPeople/searchResultSaga.js
Comment (MarcelUrbanczyk) [reply src/features/SearchResults/SearchPeople/searchResultSaga.js:1]: yup, everything was happening in original sagas

Comment (Soniq7) [reply src/features/SearchResults/SearchPeople/searchResultSaga.js:1]: ok, no probs - all good than! 
great job! :D

**Pull request merged**